tests/scripts/get_list_for_release_note.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -e
+
+# NOTE: We can skip authenticating github username and token but that will limit the github api request to 60request/IP/hr and by authenticating the limit will be 5000.
+USERNAME=""
+TOKEN=""
+
+# This is just example branch name, change according to uses
+FROM_BRANCH="upstream/release-1.6" # example branch
+TO_BRANCH="v1.6.7" # example branch
+
+
+# for minor releases
+function minor_release() {
+for pr in $(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_BRANCH} | grep pull | sed -e "s/.*#\([0-9]\+\).*/\1/g" | sort -rn | uniq); do
+	backport_prs=$(curl -u ${USERNAME}:${TOKEN} https://api.github.com/repos/rook/rook/pulls/${pr} 2>/dev/null | jq '.title';)
+	original_prs=$(echo $backport_prs |grep -E0o '[0-9]' | tr -d '\n')
+	title_with_user=$(curl -u ${USERNAME}:${TOKEN} https://api.github.com/repos/rook/rook/pulls/${original_prs} 2>/dev/null |  jq '.title+ "(#"+" " +",@"+.user.login+")"';)
+	result=$(echo "$title_with_user" | sed "s/(#/(#$original_prs/" | tail -c +2 | head -c -2)
+	echo $result
+done
+}
+
+# for major releases
+function major_release(){
+for pr in $(git log --pretty="%s" --merges --left-only ${FROM_BRANCH}...${TO_BRANCH} | grep pull | sed -e "s/.*#\([0-9]\+\).*/\1/g" | sort -rn | uniq); do
+       title_with_user=$(curl -u ${USERNAME}:${TOKEN} https://api.github.com/repos/rook/rook/pulls/${pr} 2>/dev/null |  jq '.title+ "(#"+" " +",@"+.user.login+")"';)
+       result=$(echo "$title_with_user" | sed "s/(#/(#$pr/" | tail -c +2 | head -c -2)
+       echo $result
+done
+}
+
+selected_function="$1"
```

Comment:
yeah, but for major release also we want the pr title at least, right?

Reply:
For the quarterly release, we take the release notes from the PendingReleaseNotes.md, and don't compile the list of all changes. It would be nice to generate this list though to review and help inform if anything was missed in the release notes. And perhaps we could publish it too, just not sure where yet.

Reply:
oh right, I forgot about the PendingReleaseNotes but let see if we can put the complete list somewhere.